packages/familie-header/src/Søk.less

```diff
@@ -0,0 +1,56 @@
+@inactiveColor: #78706a;
+@activeColor: #d7dae0;
+@focusColor: #ead744;
```

Comment:
Use nav-frontend colors. I know helse does not use them, but all are defined in typografi package.

Reply:
ok